src/Data/Product/Relation/Binary/Lex/Strict.agda

```diff
@@ -134,24 +134,26 @@ module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂}
     antisym (inj₂ x≈≤y)  (inj₂ y≈≤x)  =
       proj₁ x≈≤y , antisym₂ (proj₂ x≈≤y) (proj₂ y≈≤x)
 
+  ×-respectsʳ : Transitive _≈₁_ →
+                _<₁_ Respectsʳ _≈₁_ → _<₂_ Respectsʳ _≈₂_ →
+                _<ₗₑₓ_ Respectsʳ _≋_
+  ×-respectsʳ trans resp₁ resp₂ y≈y' (inj₁ x₁<y₁) = inj₁ (resp₁ (proj₁ y≈y') x₁<y₁)
+  ×-respectsʳ trans resp₁ resp₂ y≈y' (inj₂ x≈<y)  = inj₂ (trans (proj₁ x≈<y) (proj₁ y≈y')
+                                                       , (resp₂ (proj₂ y≈y') (proj₂ x≈<y)))
+  ×-respectsˡ : IsPartialEquivalence _≈₁_ →
```

Comment:
Could we split this up into separate `Symmetric` and `Transitive`  proofs? We don't tend to use partial equivalences very much in the library...